packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -174,34 +172,32 @@ export const Error = ({ label, flex, error, help, selectValue }) => {
       />
       <InputField placeholder="YYYY" error="Something went wrong on year field" help={help} />
     </InputGroup>
-  );
-};
+  ),
 
-Error.args = {
-  label: "Label",
-  flex: ["0 0 60px", "1 1 100%", "0 0 90px"],
-  error: "Something went wrong on day field",
-  help: "",
-  selectValue: selectOptionsError[0].value,
-};
+  parameters: {
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur", "onBlurGroup", "size", "error"],
```

Comment:
I'd also exclude `help` here, as there's always an error and it will never be visible.

Reply:
Excluded, as well as `helpClosable`. And I removed `error` from the list.